rules/windows/powershell/powershell_script/posh_ps_import_module_dll.yml

```diff
@@ -0,0 +1,29 @@
+title: Powershell Inport-Module Direct Dll - PsScript
+id: 9e620995-f2d8-4630-8430-4afd89f77604
+related:
+    - id: 70bc5215-526f-4477-963c-a47a5c9ebd12
+      type: similar
+status: experimental
+description: Detects use of Import-Module from a dll
+references:
+    - https://github.com/samratashok/ADModule
+    - https://twitter.com/cyb3rops/status/1617108657166061568?s=20
+author: frack113
+date: 2023/01/22
+tags:
+    - attack.reconnaissance
+    - attack.discovery
+    - attack.impact
+logsource:
+    product: windows
+    category: ps_script
+    definition: Script Block Logging must be enable
+detection:
+    selection:
+        ScriptBlockText|contains|all:
+            - 'Import-Module '
+            - '.dll'
```

Comment:
Import-Module is very common in Scripts unfortunately and due to the nature of the ScriptBlock logging there is no context. So it needs to be written as a regex to make sense.

Either way, we need to reduce the level to medium since without a specific suspicious DLL we can't guarantee it's interesting